field-agent-ui/src/components/DeleteAgent.js

```diff
@@ -0,0 +1,120 @@
+import {useContext, useEffect, useState} from "react";
+import AuthContext from "../AuthContext";
+import { Link, useHistory, useParams } from 'react-router-dom';
+
+function DeleteAgent() {
+    const [firstName, setFirstName] = useState('');
+    const [middleName, setMiddleName] = useState('');
+    const [lastName, setLastName] = useState('');
+    const [dob, setDob] = useState('');
+    const [heightInInches, setHeightInInches] = useState('');
+    const auth = useContext(AuthContext);
+    const [agents, setAgents] = useState([]);
+    const [errors, setErrors] = useState([]);
+    const { id } = useParams();
+    const history = useHistory();
```

Comment:
It's a best practice to keep all of your `useState` hooks together, preferably ahead of the `useContext`, `useParams`, and `useHistory` hooks.